velox/dwio/common/CachedBufferedInput.cpp

```diff
@@ -29,6 +29,11 @@ DEFINE_int64(
     128 << 20,
     "Maximum size of single coalesced IO");
 
+DEFINE_int32(
+    max_coalesced_distance_bytes,
+    512 << 10,
+    "Maximum distance in bytes in which coalesce will combine requests");
+
```

Comment:
Is this needed here?  Should the GFLAG be moved into the Benchmark class with the rest of the DEFINE_xxx?

Reply:
The other DEFINE for coalesce is also not needed here and should be moved

Reply:
So, what is the effect of this. They will NOT be present if the benchmarks are not built which are optional. Won't I get Linker Errors when not building the benchmarks?

Reply:
Ok tried this with no link errors. Will move it.